common/schemas/costAllocation.js

```diff
@@ -8,4 +8,4 @@ const costAllocateSchema = Joi.object({
   })
 });
 
-export default costAllocateSchema;
```

Comment:
shouldn't the prettier pre-commit hook catch these?

Reply:
@Sun-Mountain following up on this. I mis-understood the diff. I originally thought the end-of-file newline was being removed but it was being added (which is correct and prettier would take care of that)